analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -106,7 +106,7 @@ abstract class AbstractIntegrationSpec : StringSpec() {
 
                 results.size shouldBe files.size
                 results.values.forEach { result ->
-                    result.project.vcs.provider shouldBe pkg.vcs.provider
+                    result.project.vcs.provider.toLowerCase() shouldBe pkg.vcs.provider.toLowerCase()
```

Comment:
Not related to this change in particular, but it just occurred to me that we compare projects to packages here, which is wrong, or?

Regarding this change, why can we have a difference in case here after all? Could you please explains as part of the commit message?

Reply:
No that's fine, it's comparing that the VCS info parsed from the definition file after cloning is the same as the VCS info used to clone the project. That's also why the strings can be different. I have changed the implementation again to compare VCS objects now and updated the commit message.